src/arkode/arkode_bandpre.c

```diff
@@ -307,85 +305,80 @@ int ARKBandPrecGetNumRhsEvals(void *arkode_mem, long int *nfevalsBP)
 
  bp_data is a pointer to preconditoner data (set by ARKBandPrecInit)
 
- The value to be returned by the ARKBandPrecSetup function is
-   0  if successful, or
-   1  if the band factorization failed.
+ Returns a SUNLsStatus.
 ---------------------------------------------------------------*/
 static int ARKBandPrecSetup(realtype t, N_Vector y, N_Vector fy,
```

Comment:
Given the changed comment, shouldn't the function signature be changed accordingly?
```suggestion
 Returns a SUNLsStatus.
---------------------------------------------------------------*/
static SUNLsStatus ARKBandPrecSetup(realtype t, N_Vector y, N_Vector fy,
```